base/number.jl

```diff
@@ -18,9 +18,27 @@ isinteger(x::Integer) = true
 
 Return `true` if `x == zero(x)`; if `x` is an array, this checks whether
 all of the elements of `x` are zero.
+
+```jldoctest
+julia> iszero(0.0)
+true
+```
 """
 iszero(x) = x == zero(x) # fallback method
 
+"""
+    isone(x)
```

Comment:
item 3 at https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#adding-a-new-docstring-to-base, wish there was a faster way for me to link to that when I'm on mobile